minitest-display.gemspec

```diff
@@ -5,7 +5,7 @@
 
 Gem::Specification.new do |s|
   s.name = "minitest-display"
-  s.version = "0.1.0"
+  s.version = "0.1.1"
```

Comment:
Do we have `version.rb` available for this gem?

Reply:
We can check versions in github tags `git tag`

Reply:
$ git tag
0.1.0.1
v
v0.0.1
v0.0.2
v0.0.3
v0.0.4
v0.1.0

Reply:
Every gem comes with a version.rb. That's why

Reply:
Ohk, but this doesn't have any version.rb